web/src/pages/onboarding.tsx

```diff
@@ -217,7 +217,7 @@ const OnboardingPage = (props: Props): ReactElement => {
 
       const currentBusiness = currentUserData.businesses[currentUserData.currentBusinessId];
       if (currentBusiness.onboardingFormProgress === "COMPLETED") {
-        await router.replace(ROUTES.profile);
+        await router.replace(ROUTES.dashboard);
```

Comment:
updated based on [this discussion](https://njcio.slack.com/archives/C05BCE7A2KB/p1737481635280729)

Reply:
Thank you!